src/auto/unix_mount_entry.rs

```diff
@@ -1,17 +0,0 @@
-// This file was generated by gir (https://github.com/gtk-rs/gir)
-// from gir-files (https://github.com/gtk-rs/gir-files)
-// DO NOT EDIT
-
-use gio_sys;
-use gobject_sys;
-
-glib_wrapper! {
-    #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
-    pub struct UnixMountEntry(Boxed<gio_sys::GUnixMountEntry>);
-
-    match fn {
-        copy => |ptr| gobject_sys::g_boxed_copy(gio_sys::g_unix_mount_entry_get_type(), ptr as *mut _) as *mut gio_sys::GUnixMountEntry,
-        free => |ptr| gobject_sys::g_boxed_free(gio_sys::g_unix_mount_entry_get_type(), ptr as *mut _),
-        get_type => || gio_sys::g_unix_mount_entry_get_type(),
```

Comment:
Well this part could stay autogenerated but not too useful I guess :)

Reply:
I thought at start that I can use concrete functions but `g_unix_mount_copy` has `feature = "v2_54"` and I just forgot to move it back,
Agree that is right to generate even part of implementation because it can be simply expanded when gir evolves

Reply:
Hm, seems I implemented it manually because generated derives PartialEq, Eq, PartialOrd, Ord and gir seems don't have way to change it.

Reply:
See https://github.com/gtk-rs/gir/issues/828#issuecomment-522209798

Reply:
Want to do something about this one or merge for now?